src/main/java/org/gridsuite/merge/orchestrator/server/MergeOrchestratorService.java

```diff
@@ -186,13 +189,12 @@ void merge(ProcessConfig processConfig, ZonedDateTime dateTime, String date, Str
         }
     }
 
-    private List<UUID> findNetworkUuidsOfValidatedIgms(ZonedDateTime dateTime, String process) {
+    private List<IgmEntity> findNetworkUuidsOfValidatedIgms(ZonedDateTime dateTime, String process) {
```

Comment:
The method was named findUuids and used to return UUIDs. If you change the returned value, you should also rename the method

Reply:
done